### ISSUE TITLE
React Native 0.62 upgrade (Xcode) wrong image

### ISSUE THREAD

**Issue by ottob** [react-native-community/upgrade-support#21]
## Environment

n/a

## Upgrading version

0.62

## Description

The second screenshot in the guide (project localization migration) is identical to the first screenshot (organization name change).

Same thing in both https://github.com/react-native-community/upgrade-helper/issues/191 and https://github.com/react-native-community/upgrade-support/issues/13

## Reproducible demo

Browse urls above.

**Comment by ottob** [comment]
The guide says:
> Upgrade all other files except the *.xcodeproj file.

After the changes done in Xcode I still had to add sync these changes manually to the pbxproj file, according to https://react-native-community.github.io/upgrade-helper/?from=0.61.5&to=0.62.0
1. these:
<img width="1171" alt="Screenshot 2020-03-29 at 16 32 56" src="https://user-images.githubusercontent.com/67104/77851812-64bb6280-71db-11ea-9e17-d84ecbcf6699.png">
and

2. the two `LD_RUNPATH_SEARCH_PATHS = "/usr/lib/swift $(inherited)";` changes.

**Comment by Gerlison** [comment]
> The guide says:
> 
> > Upgrade all other files except the *.xcodeproj file.
> 
> After the changes done in Xcode I still had to sync these changes manually to the project.pbxproj file, according to https://react-native-community.github.io/upgrade-helper/?from=0.61.5&to=0.62.0
> 
> 1. these:
> 
> <img alt="Screenshot 2020-03-29 at 16 32 56" width="1171" src="https://user-images.githubusercontent.com/67104/77851812-64bb6280-71db-11ea-9e17-d84ecbcf6699.png">
> 
> and
> 1. the two `LD_RUNPATH_SEARCH_PATHS = "/usr/lib/swift $(inherited)";` changes.

I did not understand the second one, can you explain what changes to what?

**Comment by ottob** [comment]
> I did not understand the second one, can you explain what changes to what?

Look at https://react-native-community.github.io/upgrade-helper/?from=0.61.5&to=0.62.0
Its the changes on line 832 and 885 in ios/RnDiffApp.xcodeproj/project.pbxproj
I cannot find a way to link to those changes directly.

But I had to add those two `LD_RUNPATH_SEARCH_PATHS` lines manually to my project.pbxproj file.

**Comment by pvinis** [comment]
I will update the image soon.

**Comment by pvinis** [comment]
Fixed. Thanks for reporting.

**Comment by habibalamin** [comment]
@pvinis I don't see that this issue has been resolved. I've followed the Xcode upgrade guide for 0.62, and I get the same results as OP. Ignoring the `RnDiffApp.xcscheme` file changes, the `project.pbxproj` changes shown by the OP I also don't get when I follow all the steps. The addition of `ENABLE_BITCODE = NO`, `GCC_PREPROCESSOR_DEFINITIONS = …`, and `LD_RUNPATH_SEARCH_PATHS = …`, as well as the removal of `DEAD_CODE_STRIPPING = NO`,
